sandbox/package.json

```
@@ -16,6 +16,6 @@
     "@wdio/spec-reporter": "^9.14.0"
   },
   "dependencies": {
-    "wdio-camera-service": "file:../wdio-camera-service-1.0.0.tgz"
+    "wdio-camera-service": "file:..\\wdio-camera-service-1.0.1.tgz"
```

Now that we have pnpm we can create a `pnpm-workspace.yaml` and define the root path and `/sandbox` as workspaces so this can change to:
```suggestion
    "wdio-camera-service": "workspace:*"
```

---

I have introduced the `pnpm-workspace.yaml` but I kept the tarball approach as is, because somewhy I couldnt introduce changes to the sandbox (dummy console.log did not fire with `"workspace:*"`)

---

I wonder if it is because `'sandbox/**'` is not correct. I think you would use that if you plan to have multiple sub packages in `/sandbox` but we only have this one project in there. Can you double check if changing the workspace identifier may fix this, e.g. just suing `'sandbox/'` or something like that?

---

Ooohyeeeeaaa!!!

`sandbox/` works fine as expected! Thank you!

---

Ohhyeeeaaa!

Thank you! This works as expected with `sandbox/`